src/lib/lldpctl.h

```diff
@@ -786,6 +786,7 @@ typedef enum {
 	lldpctl_k_custom_tlvs = 5000,		/**< `(AL)` custom TLVs */
 	lldpctl_k_custom_tlvs_clear,		/** `(I,WO)` clear list of custom TLVs */
 	lldpctl_k_custom_tlv,			/** `(AL,WO)` custom TLV **/
+	lldpctl_k_custom_tlv_op,		/**< `(I,WO)` custom TLV operation */
```

Comment:
You need to put this one at the end of the "section". Otherwise, you introduce an ABI incompatibility (existing apps will break).